T2Wil/WayFarer-v2
# #167766204 API endpoint to get a specific trip

#### What does this PR do?
It creates:
a get specific trip endpoint: /trips/:id
#### Description of Task to be completed?
* Create a /trips route
* Create a getTrip controller
#### How should this be manually tested?
* have Postman
* clone the repository
* run the server/server.js  (nodemon -r esm server/server.js)
*Follow this parameter specifications to be passed in the postman:
GET  trips/trip_id;
NOTICE : trip_id with 123 is already saved in the application
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#167766204
